README.md

```diff
@@ -103,8 +103,9 @@ export default HandNote;
 
 ### Props
 1. `selectedTool` - Set the tool id to be selected.
-2. `localSourceImagePath` - Local file path of the image.
-3. `onSaveSketch(saveArgs)` - Callback when saving is complete.
+2. `penColor` - Set color for pen (currently only for Android).
```

Comment:
I would change the prop name from `penColor` to `toolColor` because I'm thinking of pen as one of the many tools with their own style of sketching. For example, a pencil tool might have the effect of pencil on paper. But that's currently not in the scope. But visualizing a tool having properties like color, thickness etc. is a good start.